### PR TITLE
Fix licenses

### LINKED ISSUE #944: Update licenses for libraries
Show licenses for all libraries (old ones are shown but new ones not) inside the app before someone flags us for violating GPL or any other license.

### PR DESCRIPTION
**Pull request by EmmanuelMess**
This adds [AboutLibraries](https://github.com/mikepenz/AboutLibraries) that helps contain in a nice UI libraries and their licenses.
-But @EmmanuelMess - you say - so many files are a fire hazard!-
There's a [pull request](https://github.com/mikepenz/AboutLibraries/pull/368) to add them to the library itself, when that is pulled and a new lib version is released I'll remove the duplicates from Amaze.

Fixes #944.

Known issues:
* The About section has no AMOLED black background. #978 
* The About section has icons that are grey 600 as opposed to the standard grey 700. #979

### CODE REVIEW

File: app/src/main/java/com/amaze/filemanager/utils/PreferenceUtils.java [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import android.graphics.Color;
 
 /**
- * Created by Vishal on 12-05-2015.
+ * Created by Vishal on 12-05-2015
+ *      edited by Emmanuel Messulam <emmanuelbendavid@gmai.com>
```
Comment (VishalNehra) [inline app/src/main/java/com/amaze/filemanager/utils/PreferenceUtils.java]: Wrong domain address

Comment (EmmanuelMess) [reply app/src/main/java/com/amaze/filemanager/utils/PreferenceUtils.java]: Like someone will care

Comment (VishalNehra) [reply app/src/main/java/com/amaze/filemanager/utils/PreferenceUtils.java]: I corrected it though

Comment (EmmanuelMess) [reply app/src/main/java/com/amaze/filemanager/utils/PreferenceUtils.java]: Thanks 😊

**Pull request merged**